tmt/utils/filesystem.py

```diff
@@ -0,0 +1,113 @@
+"""
+Utility functions for filesystem operations.
+"""
+
+import os
+import shutil
+
+import tmt.log
+from tmt._compat.pathlib import Path
+from tmt.utils import Command, GeneralError, RunError
+
+
+def _copy_tree_reflinks(
+    src: Path,
+    dst: Path,
+    logger: 'tmt.log.Logger',
+    workdir_root: Path,  # Keep signature consistent, even if not used here
```

Comment:
Is it actually used by any code in this file? I see it as a parameter, and being passed to calls to `_copy_tree_*`, but that seems to be it. A relict of the caching method? Dropping it would make the patch simpler, less changes in calls to `copy_tree()`.